nym-node-status-api/src/db/queries/testruns.rs

```diff
@@ -0,0 +1,156 @@
+use crate::{
+    db::models::{TestRunDto, TestRunStatus},
+    testruns::now_utc,
+};
+use futures_util::TryStreamExt;
+use nym_bin_common::models::ns_api::TestrunAssignment;
+use serde::Deserialize;
+use sqlx::{pool::PoolConnection, Sqlite};
+
+pub(crate) async fn get_testruns(conn: PoolConnection<Sqlite>) -> anyhow::Result<Vec<TestRunDto>> {
+    // TODO dz accept mut reference, repeat in all similar functions
+    let mut conn = conn;
+    let testruns = sqlx::query_as!(
+        TestRunDto,
+        r#"SELECT
+            id as "id!",
+            gateway_id as "gateway_id!",
```

Comment:
nit: do you have to expicitly tell sqlx that the field is not null? iirc it should be able to figure it out by itself

Reply:
not sure what you mean here, could you elaborate?

Reply:
as in do you have to be using 
```
  id as "id!",
  gateway_id as "gateway_id!",
  ...
```

as opposed to 
```
  id",
  gateway_id",
  ...
```

?

Reply:
but if it works, it works, that's good enough